src/sdl2_compat.c

```diff
@@ -5257,7 +5254,7 @@ SDL_Has3DNow(void)
         cpuid(0x80000000, a, b, c, d);
         if ((unsigned int)a >= 0x80000001) {
             cpuid(0x80000001, a, b, c, d);
-            return (d & 0x80000000) ? SDL_TRUE : SDL_FALSE;
+            return !!(d & 0x80000000);
```

Comment:
I prefer the explicit ternary instead of the `!!` syntax. It's easy to think it's a logical not if you're scanning quickly.

Reply:
OK, dropped